docs/GUIDE.md

```diff
@@ -0,0 +1,161 @@
+# Welcome to the OVAL-visualization-as-graph guide
+
+This tool generates a tree graph from ARF xml report from OpenSCAP scan.
+
+## Prerequisites
+
+### Required dependencies
+
+- **python3**
+- [lxml](https://pypi.org/project/lxml/)
+
+### Optional dependencies
+
+- includes required dependencies
+- [inquirer](https://pypi.org/project/inquirer/)
+
+## Installation
+
+### Fedora 30 and later
+
+```bash
+sudo dnf install oval-graph
+# Install inquirer, if you want have nice cli features. (optional)
+sudo pip3 install inquirer
+```
+
+### Installation via [pip](https://pypi.org/project/oval-graph/)
+
+```bash
+# If you want to install oval-graph to $HOME/.local/bin, you have to run the below command:
+pip3 install --user oval-graph
+# Install inquirer, if you want have nice cli features. (optional)
+pip3 install --user inquirer
+
+# If you want to install oval-graph globally instead, you have to run the below commands as admin, e.g. on Linux:
+sudo pip3 install oval-graph
+# Install inquirer, if you want have nice cli features. (optional)
+sudo pip3 install inquirer
+
+```
+
+### Installation from source
+
+```bash
+git clone https://github.com/OpenSCAP/OVAL-visualization-as-graph.git
+cd OVAL-visualization-as-graph
+
+# Install without features (light version)
+sudo pip3 install .
+
+# OR
+
+# Install with nice cli features
+sudo pip3 install ".[niceCli]"
+```
+
+> If you don't want to install anything, look [here](#run-from-source).
+
+## Commands
+
+- [arf-to-graph](#arf-to-graph) Generates a tree graph from ARF xml report
+- [arf-to-json](#arf-to-json) Generates a JSON from ARF xml report
+- [json-to-graph](#json-to-graph) Generates a tree graph from JSON
+
+### Arguments
+
+Run the utility with `-h` to see what optional args do.
+
+***
+
+## Usage Examples
+
+This commands consumes the rule name and the ARF file, which is standardized OpenSCAP output. You can read about generating ARF report files using OpenSCAP in the OpenSCAP User [Manual](https://github.com/OpenSCAP/openscap/blob/maint-1.3/docs/manual/manual.adoc).  
```

Comment:
That's better, but still, you can be more explicit that ARF is one of possible format for results of SCAP-compliant scanners (and I wouldn't hide that the respository contains sample ARF data).